src/inference/dev_api/cpp_interfaces/interface/ie_iplugin_internal.hpp

```diff
@@ -368,11 +368,11 @@ class INFERENCE_ENGINE_1_0_DEPRECATED INFERENCE_ENGINE_API_CLASS(IInferencePlugi
     void SetExeNetworkInfo(const std::shared_ptr<IExecutableNetworkInternal>& exeNetwork,
                            const std::shared_ptr<const ov::Model>& function);
 
-    std::string _pluginName;                            //!< A device name that plugins enables
-    std::map<std::string, std::string> _config;         //!< A map config keys -> values
-    std::weak_ptr<InferenceEngine::ICore> _core;        //!< A pointer to ICore interface
-    std::shared_ptr<ExecutorManager> _executorManager;  //!< A tasks execution manager
-    bool _isNewAPI;                                     //!< A flag which shows used API
+    std::string _pluginName;                                           //!< A device name that plugins enables
+    std::map<std::string, std::string> _config;                        //!< A map config keys -> values
+    std::weak_ptr<InferenceEngine::ICore> _core;                       //!< A pointer to ICore interface
+    std::shared_ptr<ov::threading::ExecutorManager> _executorManager;  //!< A tasks execution manager
+    bool _isNewAPI;                                                    //!< A flag which shows used API
 };
```

Comment:
This file is legacy, and will be removed in 24.0, not need to change it.

Reply:
agree

Reply:
Looks like we have removed usage of old threading classes from all plugins?
If yes, we can finally drop legacy entities in another PR

Reply:
Yes, all legacy thread-related interfaces call are removed from all plugins.  Will the removal of legacy entities be completed in 2023.3 ?

Reply:
I suppose we can remove threading legacy entities even now, because they are not used. And no needs to keep them in current master.